nemo/collections/asr/parts/perturb.py

```diff
@@ -94,17 +94,24 @@ def __init__(
     def perturb(self, data):
         snr_db = self._rng.uniform(self._min_snr_db, self._max_snr_db)
         noise_record = self._rng.sample(self._manifest.data, 1)[0]
-        noise = AudioSegment.from_file(noise_record['audio_filepath'], target_sr=data.sample_rate)
+        noise = AudioSegment.from_file(noise_record.audio_file, target_sr=data.sample_rate)
         noise_gain_db = min(data.rms_db - noise.rms_db - snr_db, self._max_gain_db)
-        logging.debug("noise: %s %s %s", snr_db, noise_gain_db, noise_record['audio_filepath'])
+        # logging.debug("noise: %s %s %s", snr_db, noise_gain_db, noise_record.audio_file)
```

Comment:
Can you fix ImpulsePerturbation in the same manner?

Reply:
Fixed